lib/matplotlib/texmanager.py

```diff
@@ -81,8 +88,13 @@ def __new__(cls):
 
     def get_font_config(self):
         ff = rcParams['font.family']
-        if len(ff) == 1 and ff[0].lower() in self._font_families:
-            self._font_family = ff[0].lower()
+        ff_val = ff[0].lower()
```

Comment:
This may have to check the length just in case `len(ff)` is 0, which I guess should, as it previously would not be caught in the `if`, fallback to `'serif'`.